migrations/repotracker_cleanup.go

```diff
@@ -0,0 +1,452 @@
+package migrations
+
+import (
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/build"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/model/version"
```

Comment:
I think that we can't depend on model/{build,task,version}.

If this is too hard, we can move the queries to model and implement this as an endpoint/splunk log thing.